### PR TITLE
Update channels docs

### PR DESCRIPTION
**Pull request by nrbnlulu**
This PR will hopefully merged after The channels PR will be merged #1407

### CODE REVIEW

**Comment by botberry** [comment]
Thanks for adding the `RELEASE.md` file!



Here's a preview of the changelog:

---

This release adds an integration with django channels.


---
Here's the preview release card for twitter:




Here's the tweet text: 
```
🆕 Release (next) is out! Thanks to Nir for the PR 👏

Get it here 👉 https://github.com/strawberry-graphql/strawberry/releases/tag/(next)
```

File: docs/integrations/channels.md [hunk 1]
```diff
@@ -0,0 +1,497 @@
...
+1. <address:port>/graphql
+2. <address:port>/graphql
+3. <address:port>/chat
+
+If you want, you can run 3 different instances of your application with different ports
+it should work the same!
+
+In tab #1 start the subscription:
+
+```gql
```
Comment (patrick91) [inline docs/integrations/channels.md:284]:
```suggestion
```graphql
```

File: docs/integrations/channels.md [hunk 2]
```diff
@@ -0,0 +1,497 @@
...
+    roomName
+    message
+    currentUser
+  }
+}
+```
+
+On tab #2 we will run `sendChatMessage` mutation:
+
+```gql
```
Comment (patrick91) [inline docs/integrations/channels.md:299]:
```suggestion
```graphql
```

**Comment by patrick91** [comment]
I'll close this since it was merged into the other PR :)

**Pull request closed without merging**